docs/credits.md

```diff
@@ -1869,6 +1869,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | style-to-object                                   | Mark                                                                      | [https://github.com/remarkablemark/style-to-object](https://github.com/remarkablemark/style-to-object)                                                                                                                      | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | stylehacks                                        | Ben Briggs                                                                | [https://github.com/cssnano/cssnano](https://github.com/cssnano/cssnano)                                                                                                                                                    | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | subscribe-ui-event                                | Hank Hsiao                                                                | [https://github.com/yahoo/subscribe-ui-event](https://github.com/yahoo/subscribe-ui-event)                                                                                                                                  | [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html)                                                         |
+| SumoLogic.Logging.Serilog                         | Sumo Logic                                                                | [https://github.com/SumoLogic/sumologic-net-appenders](https://github.com/SumoLogic/sumologic-net-appenders)                                                                                                                | [Undefined]()                                                                                                       |
```

Comment:
Confirmed as Apache 2.0. https://github.com/SumoLogic/sumologic-net-appenders/blob/main/LICENSE

I'm not sure why this isn't working.